src/nonlinear.jl

```diff
@@ -665,8 +665,10 @@ function MathProgBase.optimize!(m::PajaritoModel)
             # WARMSTART MIP FROM UPPER BOUND
             if !any(isnan,m.solution) && !isempty(m.solution)
                 if applicable(MathProgBase.setwarmstart!, getInternalModel(mip_model), m.solution)
+                    mip_warmstart = copy(m.solution)
+                    push!(mip_warmstart, m.objval)
                     # Extend solution with the objective variable
-                    MathProgBase.setwarmstart!(getInternalModel(mip_model), [m.solution, m.objval])
+                    MathProgBase.setwarmstart!(getInternalModel(mip_model), mip_warmstart)
```

Comment:
I was thinking what if the solver returns a column vector vs row vector, can it cause an issue?